charts/linkerd-control-plane/templates/destination.yaml

```diff
@@ -206,6 +206,9 @@ spec:
         - -identity-trust-domain={{.Values.identityTrustDomain | default .Values.clusterDomain}}
         - -default-opaque-ports={{.Values.proxy.opaquePorts}}
         - -enable-pprof={{.Values.enablePprof | default false}}
+        {{- if (.Values.destinationController).meshedHttp2ClientProtobuf }}
+        - --meshed-http2-client-params={{ toJson .Values.destinationController.meshedHttp2ClientProtobuf }}
```

Comment:
since this is in a yaml list of args, does that mean that it will gracefully handle json which contains spaces? or would we need to introduce quoting/escaping?